src/modules/tests/pre-test-declarations/cat-a-mod1/pre-test-declarations.cat-a-mod1.selector.ts

```diff
@@ -0,0 +1,3 @@
+import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/AM1/index';
+
+export const getcbtNumberStatus = (decs: PreTestDeclarations) => decs.DL196CBTCertNumber;
```

Comment:
I'd be inclined to change this to camelCase: `getCBTNumberStatus`. Your call though.